gui/src/components/ProcessList.svelte

```diff
@@ -59,6 +60,14 @@
   --panel-padding="0 1rem"
 >
   <div slot="actions">
+    <IconButton
```

Comment:
This is good now, but please note that #335 is going to replace these `ProcessList` action buttons with a single `...` Ellipsis button which opens a dropdown menu for actions like adding a new component, seeing workspace details, etc.

Luckily we didn't overlap on making the actual details page @BenElgar